iOS/Article/WebViewController.swift

```diff
@@ -310,6 +311,30 @@ extension WebViewController: WKNavigationDelegate {
 					let vc = SFSafariViewController(url: url)
 					self.present(vc, animated: true)
 				}
+			} else if components?.scheme == "mailto" {
+				decisionHandler(.cancel)
+				
+				guard let emailAddress = url.emailAddress else {
+					return
+				}
+				
+				if MFMailComposeViewController.canSendMail() {
+					let mailComposeViewController = MFMailComposeViewController()
+					mailComposeViewController.setToRecipients([emailAddress])
+					mailComposeViewController.mailComposeDelegate = self
+					self.present(mailComposeViewController, animated: true, completion: {})
+				} else {
+					let alert = UIAlertController(title: "Error", message: "This device cannot send emails.", preferredStyle: .alert)
```

Comment:
nitpick:

Wrapped user facing strings into `NSLocalizedString`
```suggestion
					let alert = UIAlertController(title: NSLocalizedString("Error", comment: "Error"), message: NSLocalizedString("This device cannot send emails.", comment: "Email sending failure message"), preferredStyle: .alert)
```

Reply:
Fixed in `f58ac4d`